models/model_choice.py

```diff
@@ -66,22 +66,25 @@
 
 
 def load_checkpoint(filename):
-    ''' Loads checkpoint from provided path
+    """
+    Loads checkpoint from provided path
     :param filename: path to checkpoint as .pth.tar or .pth
     :return: (dict) checkpoint ready to be loaded into model instance
-    '''
+    """
     try:
-        logging.info(f"=> loading model '{filename}'\n")
-        # For loading external models with different structure in state dict. May cause problems when trying to load optimizer
+        logging.info(f"\n=> loading model '{filename}'")
+        # For loading external models with different structure in state dict.
+        # May cause problems when trying to load optimizer
         checkpoint = torch.load(filename, map_location='cpu')
         if 'model' not in checkpoint.keys():
             temp_checkpoint = {}
-            temp_checkpoint['model'] = {k: v for k, v in checkpoint.items()}    # Place entire state_dict inside 'model' key
+            # Place entire state_dict inside 'model' key
+            temp_checkpoint['model'] = {k: v for k, v in checkpoint.items()}
             del checkpoint
             checkpoint = temp_checkpoint
         return checkpoint
     except FileNotFoundError:
-        raise FileNotFoundError(f"=> No model found at '{filename}'")
+        raise logging.critical(FileNotFoundError(f"\n=> No model found at '{filename}'"))
```

Comment:
Es-tu certain que ça fonctionne ?
De mon côté, non en tout cas
![Uploading Screenshot from 2021-10-06 12-08-04.png…]()
:

Reply:
tu me montre un message derreure mais c'est pas la bonne ligne ou tu as mit ton commentaire, je regarde sa

Reply:
Est-ce que ça fonctionne finalement?

Reply:
Not really still show the same think